.github/workflows/deploy-test.yaml

```
@@ -149,51 +142,49 @@ jobs:
+      - name: Build Playgrounds
         env:
           AXELIX_SBS_AUTH_JWT_ALGORITHM: HMAC512
           AXELIX_SBS_AUTH_JWT_SIGNING_KEY: $SANDBOX_JWT_SIGNING_KEY
         run: |
           make build-playground BUILD_ALL_PLAYGROUNDS="true"
 
-      - name: Deploy Playground Petclinic Maven SB 2
+      - name: Deploy Playground Spring Petclinic Maven SB 2
+        env:
+          APP_NAME: spring-petclinic-maven-sb-2
         uses: ./.github/actions/deploy-playground
```

actionlint (1.7.12)

[error] description is required in metadata of "Deploy Playground" action at "/home/jailuser/git/.github/actions/deploy-playground/action.yaml"

(action)